src/librustc_resolve/diagnostics.rs

```diff
@@ -93,6 +93,27 @@ mod bar {
 ```
 "##,
 
+E0254: r##"
+This error indicates that an imported symbol resolved to an already imported
+symbol.
+
+An example of this error:
+
+```
+extern crate foo; // foo is a crate that exported its own symbol
+
+use foo::*; // error, `foo::*` also resolves to `foo` already imported
```

Comment:
Unless I'm misunderstanding somehow, I think you meant `use foo` here. The following example will compile:

```rust
// crate foo
pub struct Blub;
```

```rust
// some other program
extern crate foo
use foo::*;

fn main() {
  let _blub = Blub;
}
```